src/auth.ts

```diff
@@ -93,13 +96,13 @@ const plugin = async (
     publicKey = Buffer.from('test');
     privateKey = Buffer.from('test');
   } else {
-    publicKey = fs.readFileSync(process.env.JWT_PUBLIC_KEY_PATH);
-    privateKey = fs.readFileSync(process.env.JWT_PRIVATE_KEY_PATH);
+    loadAuthKeys();
```

Comment:
This won't make any diff on prod?

Reply:
Nope, I made it compatible with adhoc scripts that I had to run to check jwt validity